trio/_tests/test_exports.py

```diff
@@ -494,9 +494,25 @@ def lookup_symbol(symbol):
                 missing.remove("__aiter__")
                 missing.remove("__anext__")
 
-        # intentionally hidden behind type guard
+            # __getattr__ is intentionally hidden behind type guard. That hook then
+            # forwards property accesses to PurePath, meaning these names aren't directly on
+            # the class.
         if class_ == trio.Path:
             missing.remove("__getattr__")
+            before = len(extra)
+            extra -= {
+                "anchor",
+                "drive",
+                "name",
+                "parent",
+                "parents",
+                "parts",
+                "root",
+                "stem",
+                "suffix",
+                "suffixes",
+            }
+            assert len(extra) == before - 10
```

Comment:
Codecov warnings here and above seem out of place. Is that just some codecov flakiness?

Reply:
It seems fine [on the Codecov site](https://app.codecov.io/gh/python-trio/trio/pull/2815?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=python-trio#c49ba6e4fd250a135d068a5402b778e7-R515), so it's probably just GitHub integration not working perfectly.